src/control/server/README.md

```diff
@@ -1,431 +1,204 @@
-# DAOS Server (control-plane)
+# DAOS Server
 
-## Workflow
+## Design
 
-Control plane server (`daos_server`) instances will open a gRPC channel to listen for requests from control plane client applications. Administrators can perform provisioning operations on network and storage hardware through the control plane [`dmg`](../cmd/dmg) management tool. Calling `dmg storage format` formats persistent storage on the server node, writes the superblock and starts the data plane.
+The server package implements the internals of the DAOS control server,
+the `daos_server` user-facing application is implemented by the
+[`daos_server`](/src/control/cmd/daos_server/README.md) package.
```

Comment:
Nit - feels like good introductory material to go in the top-level section, rather than being the start of the "Design" section